plugins/json/CMakeLists.txt

```
@@ -32,11 +32,11 @@ include(GNUInstallDirs)
 
 # PDI
 find_package(PDI REQUIRED COMPONENTS plugins)
-add_subdirectory("${CMAKE_CURRENT_SOURCE_DIR}/../../vendor/json" nlohmann_json)
+find_package(nlohmann_json 3.11.4 REQUIRED)
```

We'll have to support the version from our test images, it's available for [debian](https://packages.debian.org/bullseye/nlohmann-json3-dev), [fedora](https://packages.fedoraproject.org/pkgs/json/json-devel/index.html), [spack](https://github.com/spack/spack/blob/v0.18.1/var/spack/repos/builtin/packages/nlohmann-json/package.py) & [Ubuntu](https://packages.ubuntu.com/focal/nlohmann-json3-dev) and the oldest version is 3.7.3, but 3.9.1 if we switch to Jammy as oldest supported Ubuntu, so that's the minimal version we should require.

```suggestion
find_package(nlohmann_json 3.9.1 REQUIRED)
```